File: SDKs/amazonaws.com/ec2/2016-11-15/typescript/src/sdk/models/shared/importkeypairresult.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class ImportKeyPairResult extends SpeakeasyBase {
  @SpeakeasyMetadata()
  keyFingerprint?: Record<string, any>;

  @SpeakeasyMetadata()
  keyName?: Record<string, any>;

  @SpeakeasyMetadata()
  keyPairId?: Record<string, any>;

  @SpeakeasyMetadata()
  tags?: Record<string, any>;
}

File: SDKs/amazonaws.com/ec2/2016-11-15/python/src/sdk/models/shared/importkeypairresult.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class ImportKeyPairResult:
    key_fingerprint: Optional[dict[str, Any]] = dataclasses.field(default=None)
    key_name: Optional[dict[str, Any]] = dataclasses.field(default=None)
    key_pair_id: Optional[dict[str, Any]] = dataclasses.field(default=None)
    tags: Optional[dict[str, Any]] = dataclasses.field(default=None)
    
